srcpkgs/transcode/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'transcode'
+pkgname=transcode
+version=1.1.7
+revision=1
+build_style=gnu-configure
+hostmakedepends="pkg-config"
+makedepends="zlib-devel ffmpeg-devel libdvdread-devel libjpeg-turbo-devel
+ libX11-devel libXaw-devel libXpm-devel libXv-devel libmpeg2-devel freetype-devel
+ xvidcore-devel libogg-devel libvorbis-devel libtheora-devel lzo-devel SDL2-devel
+ libxml2-devel faac-devel"
+short_desc="Modular utility for video stream processing"
+maintainer="Andrew Benson <abenson+void@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://bitbucket.org/france/transcode-tcforge"
+distfiles="https://bitbucket.org/france/transcode-tcforge/downloads/transcode-${version}.tar.bz2"
+checksum=1e4e72d8e0dd62a80b8dd90699f5ca64c9b0cb37a5c9325c184166a9654f0a92
+
+if [ -n "$CROSS_BUILD" ]; then
+	configure_args+=" --disable-lame"
```

Comment:
Why is lame disabled for cross builds? AFAICT it is available for all architectures.

Reply:
I'll see if I can disable the test program it builds

Reply:
Ok, configure scripts running test programs without checking for cross builds are troublesome.
I didn't know this was the reason to disable lame.
Since Travis is failing for libsearpc-3.0.8, which doesn't exist(?), I'll take a look myself.

Reply:
I was asking about that in IRC this morning -- there is a PR for 3.0.8 but it's not been merged and I can't find it having been committed anywhere. Really confused on that one.